<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MTX Editor</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="g1_min">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>600</height>
       </size>
      </property>
      <property name="title">
       <string>Minimum</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDoubleSpinBox" name="g1_x">
         <property name="prefix">
          <string/>
         </property>
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="g1_y">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="g1_z">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="g2_max">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>600</height>
       </size>
      </property>
      <property name="title">
       <string>Maximum</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QDoubleSpinBox" name="g2_x">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="g2_y">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="g2_z">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="g3_cog">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>600</height>
       </size>
      </property>
      <property name="title">
       <string>Center of gravity</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QDoubleSpinBox" name="g3_x">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="g3_y">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="g3_z">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="g4_origin">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>600</height>
       </size>
      </property>
      <property name="title">
       <string>Origin</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="g4_x">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="g4_y">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="g4_z">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-500000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>500000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="buttonOpen">
           <property name="text">
            <string>Open</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="buttonSpec">
           <property name="text">
            <string>Format specification</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::MailAttachment"/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="buttonSave">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="buttonAbout">
           <property name="text">
            <string>About</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>469</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSpec"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSpec">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MailAttachment"/>
   </property>
   <property name="text">
    <string>&amp;MTX format specification</string>
   </property>
   <property name="toolTip">
    <string>See the specification of the MTX format on the Web</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>&amp;About this program</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Close the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create a new MTX file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOpen</sender>
   <signal>clicked()</signal>
   <receiver>actionOpen</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSpec</sender>
   <signal>clicked()</signal>
   <receiver>actionSpec</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>actionSave</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAbout</sender>
   <signal>clicked()</signal>
   <receiver>actionAbout</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
